### DISCUSSION TITLE
Multiple Repo Best Practices?

### DISCUSSION THREAD

**Discussion by JobiOne42** [community/community#42787] [Repositories] [answered]
### Select Topic Area

Question

### Body

I'm currently working on two separate projects for work and have two repos in GitHub and like to use ssh to commit. What I've been doing is changes my origin remote back and forth every time I have to commit to the other project. I also saw however I could add multiple remotes, but it kind of seems like everywhere I look this is referenced for upstream main project and the same project somewhere else. 

Is there a standard or best practice for this?

**Comment by airtower-luna** [comment, marked as answer]
Using multiple remotes is the right thing if you work with multiple remote repositories *of the same project*, see: [Git Basics - Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes)

However in your post you mention one being the "other" project. If they are separate you should be using separate *local* repositories, just clone both remote repositories to different local ones.

**Comment by Weifeng-Chen** [comment]
【自动恢复】来信已收到，我将尽快回复你！
